### ISSUE TITLE
Fingerprint

### ISSUE THREAD

**Issue by WevoDex** [CheshireCaat/selenium-with-fingerprints#23]
I have my own fingerprint db how can i setting up ?

**Comment by CheshireCaat** [comment]
You can use fingerprints from the database, just specify the service key if they obtained using the premium version, or leave the key empty if they are free.

**Comment by WevoDex** [comment]
I created my own API for fingerprinting. Basically, I have a bunch of real fingerprints that I want to use. I noticed that Bablosoft's API is similar to mine, but it's quite expensive. Is there a way to use my own fingerprints instead of Bablosoft's?

Here are the parameters I currently have:

userAgent
resolution
language
platform
canvas
hardwareConcurrency
deviceMemory
maxTouchPoints
mediaDevices
webGLMetadata
webglParams
webGL
clientRects
devicePixelRatio
fonts list
browserType
Additionally, I have my own compiled version of Chrome to reduce detection, which is based on the original Chromium.

Are there any options to set up my own fingerprinting system using the parameters I have?
